lib/src/pages/auth/widgets/common/custom_button_widget.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class CustomButtonWidget extends StatelessWidget {
+  Widget childWidget;
+  Function() onPressed;
+  Color? color = Color(0XFF3772FF);
+  CustomButtonWidget({
+    Key? key,
+    required this.childWidget,
+    required this.onPressed,
+    this.color,
+  }) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      onPressed: () {},
+      child: childWidget,
+      style: ElevatedButton.styleFrom(
+        primary: color,
+        fixedSize: Size(340, 55),
```

Comment:
Keeping this as a fixed size would probably make it look wrong on some screens. You might want to change it with something like the following:
```suggestion
        fixedSize: Size(Get.size.width * 0.9, 55),
```
This would make it so that the button is always exactly 90% of the **screen width**